Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -3389,28 +3391,35 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Geom' : 'ExtendedRun4D116',
         'HLTmenu': '@relvalRun4',
         'GT' : 'auto:phase2_realistic_T33',
-        'Era' : 'Phase2C17I13M9',
+        'Era' : 'Phase2C22I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
     },
     'Run4D117' : {
         'Geom' : 'ExtendedRun4D117',
         'HLTmenu': '@relvalRun4',
         'GT' : 'auto:phase2_realistic_T33',
-        'Era' : 'Phase2C17I13M9',
+        'Era' : 'Phase2C22I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
     },
     'Run4D118' : {
         'Geom' : 'ExtendedRun4D118',
         'HLTmenu': '@relvalRun4',
         'GT' : 'auto:phase2_realistic_T33',
-        'Era' : 'Phase2C17I13M9',
+        'Era' : 'Phase2C22I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
     },
     'Run4D119' : {
         'Geom' : 'ExtendedRun4D119',
         'HLTmenu': '@relvalRun4',
         'GT' : 'auto:phase2_realistic_T33',
-        'Era' : 'Phase2C17I13M9',
+        'Era' : 'Phase2C22I13M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
+    },
+    'Run4D120' : {
+        'Geom' : 'ExtendedRun4D120',
+        'HLTmenu': '@relvalRun4',
+        'GT' : 'auto:phase2_realistic_T33',
+        'Era' : 'Phase2C22I13M9',
```

Comment:
Unrelated to this PR (already signed) but why we have naming for Phase2 eras that does not sound like they correspond to the actual detector setup. E.g. here `Phase2C22I13M9` suggest `C22+I13+M9` but `D120` is different. Is it that there's no needed modification (in terms of era) from  `C22+I13+M9` to  `C26+I20+M15`?